vendor/engines/split_accounts/app/models/split_accounts/split_account.rb

```diff
@@ -27,5 +29,16 @@ def extra_penny_count
       splits.select(&:extra_penny?).size
     end
 
+    # Stopped using SQL because that didn't seem to work until the built splits
+    # and subaccounts were created.
+    #
+    # SQL version:
+    #   subaccounts.where("expires_at IS NOT NULL").order(expires_at: :asc).first
+    #
+    def earliest_expiring_subaccount
+      subaccounts = splits.map{ |split| split.subaccount }.select(&:expires_at)
```

Comment:
The `has_many` `through` was not working until I created the `splits` in the database.  I spent some time trying to find a workaround but was unsuccessful.  Is this a rails 3.2 issue or is that expected rails behavior?  At the end of the day, I'd like to use this method on a non-persisted `split_account` (self) with non-persisted `splits` (the through `subaccounts` are persisted).